addon/index.js

```diff
@@ -7,8 +7,9 @@ Ember.Registry.prototype.normalize = function(i) { return i; }
 
 export default class Resolver {
   constructor(attrs) {
-    this.namespace = attrs.namespace;
-    this.modulePrefix = this.namespace.modulePrefix;
+    if (attrs) {
+      this.namespace = attrs.namespace;
+    }
```

Comment:
The tests harness uses test-helpers/resolver.js. For some reason, the namespaces, is set AFTER create